app/src/main/res/values/strings.xml

```diff
@@ -399,4 +399,11 @@
     <string name="metric_newest">\"SUSI, what are the newest skills?\"</string>
     <string name="metrics_top_games">\"SUSI, what are your top games?\"</string>
     <string name="message_no_skills_found">No skills found.</string>
+
+    <string name="report_skill">Report Skill</string>
+    <string name="report_send">Send Report</string>
+    <string name="report_error">Coudn\'t report the skill</string>
+    <string name="skill_already_reported">Skill already reported</string>
```

Comment:
please use proper naming conventions. In most of the cases, you have used the message as the key name. You could say 'message_report_skill', 'error_report_skill' etc. Please check it for other added strings as well.